apps/code-sketch/code-sketch.yml

```diff
@@ -0,0 +1,28 @@
+name: 'Code Sketch'
+description: 'First place where the code was written...'
```

Comment:
The App gallery's pretty loose on what descriptions are allowed; but even so, this description doesn't explain what the app is at all. Could you make this more descriptive of what the app does / what it's used for?

Reply:
OK，let me think about another desc 🤔